### DISCUSSION TITLE
Snort in SO

### DISCUSSION THREAD

**Discussion by Jackson-Pollock** [Security-Onion-Solutions/securityonion#5137] [Unsupported Versions] [answered]
As far as I know SO replaced snort with suricata but this docs says about snort rule updation. https://docs.securityonion.net/en/2.3/rules.html 
Can we still utilize snort rules in SO?

**Comment by gcsrellis** [comment, marked as answer]
Snort rules will work.  Suricata can use the snort rule set and the ET ones.

**Comment by dougburks** [comment]
As described in https://docs.securityonion.net/en/2.3/rules.html, Suricata can load standard Snort rules but please note that some rules in the Snort rulesets are Shared Object rules which will not load in Suricata.

**Reply by petiepooo** [reply]
In the interest of making information easy to find, the shared-object in the Talos/registered ruleset can be disabled in SO2 by ensuring that your minion state file includes the following regex under idstools: / sids: / disabled:

> \- 're:soid [0-9]+'

That does mean you'd be missing out on quite a few of the ruleset's features, though, which is why I'm still hoping SOS will return to giving us an option to run Snort v3...  Cisco's criteria for oem licensing their ruleset is much simpler/cheaper than Proofpoint's licensing of their ET Pro ruleset.

**Reply by dougburks** [reply]
Added to documentation:
https://docs.securityonion.net/en/2.3/rules.html#snort-registered
